src/dag.ts

```diff
@@ -0,0 +1,88 @@
+import { Model } from './model';
+import { Script } from './script';
+import { Tensor } from './tensor';
+
+export interface DagCommandInterface {
+  tensorset(keName: string, t: Tensor): DagCommandInterface;
```

Comment:
Mybe we should change `keName` to `keyName` and keep consistent with other places？

Reply:
agree. Fixed in the latest commit